cucumber/api/features/support/ssh_helpers.rb

```diff
@@ -0,0 +1,16 @@
+# frozen_string_literal: true
+
+require 'net/ssh'
+
+# Utility methods for performing actions in remote machines
+module SshHelpers
+
+  def run_command_in_machine(machine_ip, machine_username, machine_password, command)
+    ssh = Net::SSH.start(machine_ip, machine_username, :password => machine_password)
+    res = ssh.exec!(command)
+    ssh.close
+    res
```

Comment:
I imagine we can use .tap here instead of just returning `res`

Reply:
Done